### PR TITLE
Update layout login view for isis

### PR DESCRIPTION
**Pull request by n9iels**
## The problem

I discovered that the alignment on the login page of isis is not 100% pixel perfect. The field are just a few pixels to width. Hower, on RTL languages the wrong alignment was very good visible. So this patch solves alignment problems on LTR and RTL languages. I also add the possibility to load the `templates-rtl.css` in the login-view.
Farther optimizations:
- font-set icon `icon-joomla` instead of .png icon for the Joomla! logo on bottom of the loginform
- Full width login button
## Before/after screenshots

<b>LTR before</b>
lrt-before

<b>LTR after (very small difference on the right of the fields)</b>
ltr after

<b>RTL before</b>
rtl before

<b>RTL after</b>
rtl after
## How to test this patch
1. Confirm the before/after screenshots on a LTR language (English)
2. Confirm 100% page width button and sharp Joomla! logo on bottom
3. Change your language to a RTL language (Persion for example)
4. Test step 2 and 3 again.
5. Confirm the login layout is also good shown on mobile devices. Feel free to use the Google Chrome F12 developer tools for this part
## Notes
- Also add some extra spaces for CS on some places
- As idea of @dgt41 the login button is full-width. Please let me know how you're thoughts are about that

### CODE REVIEW

**Comment by dgt41** [comment]
Nice one @n9iels 
Do you think you could also change the button to be `btn-block` or in plain english side to side? Just a thought

**Comment by n9iels** [comment]
Good idea! I think I like it, what about you?
I also edit the test instructions :-)

**Comment by n9iels** [comment]
oops, looks like something went wrong...

**Comment by n9iels** [comment]
Can somebody help me solve the merge conflicts when testing is done?

**Comment by roland-d** [comment]
@n9iels I can help you with that, no problem.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6432">issues.joomla.org/joomla-cms/6432</a>.</sub>

**Comment by n9iels** [comment]
@roland-d thanks!
I also changed Joomla! png icon below the login fields to the font-set icon `icon-joomla`.

**Comment by credenceweb** [comment]
@test I have reproduced the problem, and this patch does fix it!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6432">issues.joomla.org/joomla-cms/6432</a>.</sub>

**Comment by matrikular** [comment]
@test OK - tested with Hebrew and Persian in Firefox and IE.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6432">issues.joomla.org/joomla-cms/6432</a>.</sub>

**Comment by dgt41** [comment]
@test OK 
Nice one @n9iels

**Comment by brianteeman** [comment]
thanks for testing - setting RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6432">issues.joomla.org/joomla-cms/6432</a>.</sub>

**Comment by wilsonge** [comment]
Sorry you're going to need to merge in staging again here. There are still conflicts!

**Comment by n9iels** [comment]
I made a redo of this PR, please test #6535

**Comment by n9iels** [comment]
@credenceweb @dgt41 @matrikular can you please test #6535? Or can @brianteeman set that PR to RTC because it is exactly the same code for a new version of the template css?
